gasCosts/gasCosts.js

```diff
@@ -110,56 +110,94 @@ contract("All", accounts => {
     });
 
     it("when working with a Task", async () => {
-      await colony.makeTask(SPECIFICATION_HASH, 1);
-      await colony.setTaskDomain(1, 1);
-      await colony.setTaskSkill(1, 7);
-      await colony.setTaskRoleUser(1, EVALUATOR_ROLE, EVALUATOR);
-      await colony.setTaskRoleUser(1, WORKER_ROLE, WORKER);
+      const { logs } = await colony.makeTask(SPECIFICATION_HASH, 1);
+      const taskId = logs[0].args.id.toNumber();
+
+      // setTaskDomain
+      await executeSignedTaskChange({
+        colony,
+        functionName: "setTaskDomain",
+        taskId,
+        signers: [MANAGER],
+        sigTypes: [0],
+        args: [taskId, 1]
+      });
```

Comment:
Same as this comment https://github.com/JoinColony/colonyNetwork/pull/214#discussion_r195729747 
Applicable to the rest of the JS changes in this PR

Reply:
For this particular example, i formated it this way, it was more readable. And for https://github.com/JoinColony/colonyNetwork/pull/214#discussion_r195729747, it was `prettier` that gave me an error.

Reply:
I personally prefer the one line formatting. But this is a topic for a different conversation that I'm happy to have outside of this PR. I don't think we should partially change the formatting here, for now just follow the one-line style which is what we've used in the remainder of the codebase.